src/components/anti-everyone.ts

```diff
@@ -34,6 +39,18 @@ export default class AntiEveryone extends BotComponent {
             await message.reply({
                 content: `Did you really just try to ping ${memberCount} people?`,
             });
+            if(!replies.has(message.author.id) replies[message.author.id] = [];
+            replies[message.author.id].push(message);
         }
     }
 }
+
+/**
+ * Deletes all Replies that were made to a particular user
+ * @note this should be used to auto-hide spam message replies in order to try and reduce the effect of spam
+ * @TODO: Actually bind this into the anti-spam system
+ */
+export function deleteReplies(user: Discord.User) {
```

Comment:
Please make the `replies` map be made members of the bot component class (components can access other components with `(unwrap(this.wheatley.components.get("TheComponent")) as TheComponent)` which is clunky to write but at the moment it's the de-facto way to go about this for the bot)